bin/wc-latest-version.php

```diff
@@ -0,0 +1,56 @@
+<?php
+function version_compare_reverse($i, $j) {
+	 return version_compare($i, $j) * -1;
+}
+
+/**
+ * Take input from stdin and adds all valid (ie. x.y.z) versinos to an array.
+ */
+function filter_valid_versions() {
+	$versions = [];
+	$fp = fopen("php://stdin", 'r');
+	$contents = fgets($fp);
+	while ($contents !== false) {
+		$contents = fgets($fp);
+		if (!preg_match('/^\d+\.\d+\.\d+$/', $contents)) {
+			continue;
+		}
+		$versions[] = trim($contents);
+	}
+	return $versions;
+}
+
+/**
+ * Take an array of $versions and return the latest 3 minor versions.
+ *
+ * @param Array $versions The array need to be sorted first.
+ * @return 3 minor versions, in the order of oldest to newest.
+ */
+function get_last_three_minor($versions) {
+	$count = 0;
```

Comment:
Looks like `$count` is no needed, since you're already doing `count($results)`, correct?

Reply:
Good catch! Previously I was using `$count` before I refactored. Removing this.